plugins/module_utils/rds.py

```diff
@@ -492,6 +502,47 @@ def wait_for_status(client, module: AnsibleAWSModule, identifier: str, method_na
         wait_for_cluster_snapshot_status(client, module, identifier, waiter_name)
 
 
+def get_snapshot(
```

Comment:
it would be nice to remove the `module` argument from this function since the caller should be able to catch the `AnsibleRDSError` exception